app/app.js

```diff
@@ -33,6 +33,13 @@ const AppFactory = function() {
         logger.error("Uncaught exception", err.message);
       });
 
+      process.on( "message", ( msg ) => {
+        if ( msg && typeof msg === "string" && msg === "quit" ) {
+          logger.info("Rise Cache is terminating");
```

Comment:
I think we should be more specific for the sake of clarity:
`Quit process message received. Rise Cache will terminate.`